README.md

```diff
@@ -1,2 +1,14 @@
 # chromeback
 Mapping of Chrome APIs to node-style callbacks
+
+## Usage
+```
+var chromeApi = require('chromeback');
```

Comment:
by exposing `buildApi`, this would become `var chromeApi = require('chromeback')(chrome)`